services/dashboard/app/routes/__root.tsx

```diff
@@ -496,3 +496,49 @@ function Tabs() {
     </nav>
   );
 }
+
+function EllipsisMenu() {
+  const [open, setOpen] = useState(false);
+  const matches = useMatches();
+  const router = useRouter();
+
+  // Determine if current location is company-performance and get investment name.
+  const companyMatch = matches.find((m) => m.pathname.includes("/investments"));
+  const investmentName = (companyMatch?.params as any)?.name;
+  const isCompany = Boolean(companyMatch);
+
+  const toggle = () => {
+    if (isCompany) setOpen(!open);
+  };
+
+  const handleTemplateConfig = () => {
+    if (!investmentName) return;
+    router.navigate({
+      to: "/investments/$name/template-config",
+      params: { name: investmentName },
+      replace: true,
+    });
+    setOpen(false);
+  };
+
+  return (
+    <div className="relative">
+      <button
+        onClick={toggle}
+        className="p-1 rounded-md hover:bg-neutral-200 dark:hover:bg-neutral-700 text-neutral-700 hover:text-neutral-900 dark:text-neutral-300 dark:hover:text-white cursor-pointer"
+      >
+        <Ellipsis className="h-4 w-4" />
+      </button>
+      {open && (
+        <div className="absolute right-0 mt-2 w-48 bg-white dark:bg-neutral-800 border border-neutral-200 dark:border-neutral-700 rounded shadow-md z-50">
+          <button
+            onClick={handleTemplateConfig}
+            className="block w-full text-left px-4 py-2 text-sm hover:bg-neutral-100 dark:hover:bg-neutral-700"
+          >
+            Template Config
+          </button>
+        </div>
+      )}
+    </div>
+  );
+}
```

Comment:
We need to make this dynamic, so its only loaded on campaigns routes. Lets do that today or tmw pair programming!